specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/CommonDefinitions.json

```diff
@@ -1228,6 +1228,214 @@
+              "$ref": "#/definitions/HttpRetryPolicy"
+            },
+            "tcpRetryPolicy": {
+              "description": "Policy that defines tcp request retry conditions",
+              "$ref": "#/definitions/TcpRetryPolicy"
+            },
+            "circuitBreakerPolicy": {
+              "description": "Policy that defines circuit breaker conditions",
+              "$ref": "#/definitions/CircuitBreakerPolicy"
+            },
+            "httpConnectionPool": {
+              "description": "Defines parameters for http connection pooling",
+              "$ref": "#/definitions/HttpConnectionPool"
+            },
+            "tcpConnectionPool": {
+              "description": "Defines parameters for tcp connection pooling",
+              "$ref": "#/definitions/TcpConnectionPool"
+            }
+          },
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "TimeoutPolicy": {
+      "description": "Policy to set request timeouts",
+      "type": "object",
+      "properties": {
+        "responseTimeoutInSeconds": {
+          "description": "Timeout, in seconds, for a request to respond",
+          "format": "int32",
+          "type": "integer"
+        },
+        "connectionTimeoutInSeconds": {
+          "description": "Timeout, in seconds, for a request to initiate a connection",
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "HttpRetryPolicy": {
+      "description": "Policy that defines http request retry conditions",
+      "type": "object",
+      "properties": {
+        "maxRetries": {
+          "description": "Maximum number of times a request will retry",
+          "format": "int32",
+          "type": "integer"
+        },
+        "retryBackOff": {
+          "description": "Settings for retry backoff characteristics",
+          "type": "object",
+          "properties": {
+            "initialDelayInMilliseconds": {
+              "description": "Initial delay, in milliseconds, before retrying a request",
+              "format": "int64",
+              "type": "integer"
+            },
+            "maxIntervalInMilliseconds": {
+              "description": "Maximum interval, in milliseconds, between retries",
+              "format": "int64",
+              "type": "integer"
+            }
+          },
+          "x-ms-client-flatten": true
+        },
+        "matches": {
+          "description": "Conditions that must be met for a request to be retried",
+          "type": "object",
+          "properties": {
+            "headers": {
+              "description": "Headers that must be present for a request to be retried",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/HeaderMatch"
+              }
+            },
+            "httpStatusCodes": {
+              "description": "Additional http status codes that can trigger a retry",
+              "type": "array",
```

Comment:
lack of x-ms-identifiers: https://github.com/Azure/azure-openapi-validator/blob/main/docs/xms-identifier-validation.md